init/init.cpp

```diff
@@ -64,6 +64,8 @@
 #include "ueventd.h"
 #include "watchdogd.h"
 
+#define SYSTEMD_SELINUX
```

Comment:
So this is always on even though SELinux is not on?

Reply:
Yes, but it does no harm either way.